examples/00-mapdl-examples/lathe_cutter.py

```diff
@@ -0,0 +1,432 @@
+"""
+.. _ref_lathe_cutter_example:
+
+=====================================
+Structural Analysis of a Lathe Cutter
+=====================================
+
+**Summary**: Basic walk through demonstration of PyMAPDL capabilities.
+
+Objective
+=========
+
+The objective of this example is to highlight some of the often-used
+PyMAPDL categories via a lathe cutter finite element model. Lathe
+cutters have multiple avenues of wear and failure, and the analyses
+supporting their design would most often be transient
+thermal-structural. However, for simplicity, we will simulate this with
+a non-uniform load.
+
+.. figure:: ../../../_static/lathe_cutter_model.png
+    :align: center
+    :width: 600
+    :alt:  Lathe cutter geometry and load description.
+    :figclass: align-center
+
+    **Figure 1: Lathe cutter geometry and load description.**
+
+
+-  `OS <https://docs.python.org/3/library/os.html>`__ is used for Python
+   method to get the current working directory
+
+-  `Numpy <https://numpy.org/>`__ is used for Numpy arrays and functions
+
+Contents
+=========
+
+1. **Launch and variables:**
+   Define necessary variables and launch MAPDL.
+
+2. **Geometry, mesh and MAPDL parameters:**
+   Import geometry and inspect for MAPDL parameters. Define linear
+   elastic material model with Python variables. Mesh and apply symmetry
+   boundary conditions.
+
+3. **Coordinate system and load application:**
+   Create a local coordinate system for applied load and verify with
+   plot.
+
+4. **Pressure load application:**
+   Define the pressure load as a sine function of the length of the
+   application area using numpy arrays; import the pressure array into
+   MAPDL as a table array; verify the applied load, and solve.
```

Comment:
```suggestion
   Define the pressure load as a sine function of the length of the
   application area using numpy arrays. Import the pressure array into
   MAPDL as a table array. Verify the applied load and solve.
```

Reply:
The earlier sections used short, imperative sentences--which is easier to read than semicolons. What you had wasn't incorrect. I just wanted it to be consistent--and avoiding the use of semicolons is recommended for technical doc.